src/aura/STG_CMP_Addr/STG_CMP_Addr.cmp

```diff
@@ -73,10 +74,13 @@
                     <label class="slds-checkbox">
                         <aura:if isTrue="{!v.isView}">
                             <ui:outputCheckbox value="{!v.hierarchySettings.Disable_Preferred_Phone_Enforcement__c}"
-                                               class="contact-addr-enabled" />
+                                               class="contact-addr-enabled"
+                                               aura:id="disablePhoneErrors" />
                         <aura:set attribute="else">
                             <ui:inputCheckbox value="{!v.hierarchySettings.Disable_Preferred_Phone_Enforcement__c}"
-                                              class="contact-addr-enabled" />
+                                              class="contact-addr-enabled"
+                                              aura:id="disablePhoneErrors"
+                                              disabled="{!v.prefPhoneErrorsDisabled}"/>
```

Comment:
@lifewithryan @robertbwatson @xinnanlisf I think this is not working as expected for the first time when the page loads. Can you guys quickly test this on the first page load as well please, just want to make sure that it's not just some caching issue with my lightning. If so was wondering if we could change it to disabled="{!not(v.hierarchySettings.Enable_New_Preferred_Phone_Sync__c)}"

Reply:
@skosgi1 Yep, thanks! Ryan and I just fixed this and will push up a commit in a few minutes with the final changes.